### PR TITLE
DB/Quest: Fall From Grace

### PR DESCRIPTION
**Pull request by CookieMonsterDev**
##### CHANGES PROPOSED:

-  Import SQL's

###### ISSUES ADDRESSED:

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1464

##### TESTS PERFORMED:

Tested on local Windows machine and works as intended after applying my SQL's.

##### HOW TO TEST THE CHANGES:

Simply import SQL's before starting up your server, or restart server after importing SQL. The SMART AI function will not work after using .reload unless you delete the creature and spawn it again.

1- .go -10632 -3009 29 0
2- Speak with NPC "Fallen Hero of the Horde"
3- Accept the only quest you can accept, called "Fallen From Grace"
4- Speak again with this NPC and click: Why are you here? > Continue with your story. > Tragic..
5. Quest will complete and close gossip window.

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1550101831834633300.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+(841,0,0,"Continue with your story.",3601,1,1,842,0,0,0,"",0,0),
+(842,0,0,"Tragic...",3830,1,1,0,0,0,0,"",0,0);
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=7572;
+INSERT INTO `smart_scripts` VALUES (7572, 0, 0, 2, 62, 0, 100, 1, 842, 0, 0, 0, 0, 26, 2784, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 0 Selected - Quest Credit \'Fall From Grace\'');
+INSERT INTO `smart_scripts` VALUES (7572, 0, 1, 3, 62, 0, 100, 1, 881, 0, 0, 0, 0, 26, 2801, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 1 Selected - Quest Credit \'A Tale of Sorrow\'');
+INSERT INTO `smart_scripts` VALUES (7572, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 0 Selected - Close Gossip');
+INSERT INTO `smart_scripts` VALUES (7572, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 1 Selected - Close Gossip');
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1550101831834633300.sql]: Add a new line at the end of the file, please.

Comment (CookieMonsterDev) [reply data/sql/updates/pending_db_world/rev_1550101831834633300.sql]: There you go, why is that needed?

Comment (Nefertumm) [reply data/sql/updates/pending_db_world/rev_1550101831834633300.sql]: Consistency.

File: data/sql/updates/pending_db_world/rev_1550101831834633300.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1550101831834633300.sql]: can you please merge those inserts? Something like:

```
INSERT INTO `smart_scripts` VALUES
(7572, 0, 0, 2, 62, 0, 100, 1, 842, 0, 0, 0, 0, 26, 2784, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 0 Selected - Quest Credit \'Fall From Grace\''),
(7572, 0, 1, 3, 62, 0, 100, 1, 881, 0, 0, 0, 0, 26, 2801, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 1 Selected - Quest Credit \'A Tale of Sorrow\''),
(7572, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 0 Selected - Close Gossip'),
(7572, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 1 Selected - Close Gossip');
```

**Comment by CookieMonsterDev** [comment]
There are two because my first PR got messed up so I copied my fix out. Must’ve forgotten about the version. I’ll fix it. Thanks for the catch!

**Comment by comix1988** [comment]
the quest now works, after listening the story from the npc, it's completed and can be delivered.

**Comment by FrancescoBorzi** [comment]
Good job @CookieMonsterDev and thanks @comix1988 @Stoabrogga @Nefertumm for testing/reviewing this PR

**Comment by BarbzYHOOL** [comment]
Awesome

**Comment by StygianTheBest** [comment]
Nice to see a lot of quest reports pouring in. For this particular quest, further testing should be done to see if the Warrior starter quests that lead to this work as well as all of the other quests that make up this EPIC classic quest chain. Truly one of the best questing experiences and a hidden gem that awarded monster XP. Another casualty of the care-bear Cataclysm changes.

Reference:
- https://wow.gamepedia.com/Fallen_Hero_of_the_Horde

**Pull request merged**